tests/test_filter_tasks.py

```diff
@@ -0,0 +1,10 @@
+import pytest
+import playground.__main__ as filter_file
+
+
+@pytest.mark.parametrize(("file_name", "sub_str", "assert_equal"), [
```

Comment:
```suggestion
@pytest.mark.parametrize(("file_name", "sub_str", "expected_result"), [
```
I think this will be more convenient variable name